includes/helpers/class-app.php

```diff
@@ -313,20 +313,20 @@ public static function array_to_html_params( $atts ) {
 	/**
 	 * @since 5.9.2
 	 */
-	public static function show_logo( $size ) {
-		echo self::kses( self::svg_logo( $size ), 'all' ); // WPCS: XSS ok.
+	public static function show_logo( $size, $extra_class = false, $round = false ) {
```

Comment:
It looks like it would be better to add a new function for the round version, instead of the extra complexity with all the arguments.

Reply:
Done